src/isbrowser.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function isBrowser(): boolean {
```

Comment:
Do we need a function for this?  Isn't it really just saying `window ?`

Reply:
In this particular case it's just saying `window ?`, but we have this function in other libraries. You can say YAGNI and I'll change it to `window ?`, but I'd rather leave it as a separate function to match other repos :)